src/main/java/io/github/bedwarsrel/BedwarsRel/Game/RessourceSpawner.java

```diff
@@ -110,7 +113,38 @@ public void setGame(Game game) {
 
   @Override
   public void run() {
-    Location dropLocation = this.location;
+    Location dropLocation = this.location.clone();
+    ItemStack item = this.itemstack.clone();
+
+    if (Main.getInstance().getBooleanConfig("spawnRessourcesInChest", true)) {

```

Comment:
We've always been using dash notation instead of camel-case in the config before. Should be changed.